latest/docs-ref-conceptual/release-notes-azure-cli.md

```diff
@@ -16,6 +16,66 @@ ms.assetid: ce0428f7-0a59-4e72-9237-d907b171af51
 
 # Azure CLI 2.0 release notes
 
+## September 22, 2017
+
+Version 2.0.18
+
+### Resource
+
+* Added support for showing built-in policy definitions
+* Added support mode parameter for creating policy definitions
+* Added support for UI definitions and templates to `managedapp definition create`
+* BREAKING CHANGE: managedapp: Update to latest ARM package, which includes changing resource type from appliances to applications and applianceDefinitions to applicationDefinitions
+
+### Network
+
+* Added support for availability zone to `network lb` and `network public-ip` subcommands
+* Added support for IPv6 Microsoft Peering to `express-route`
+* Added `asg` application security group commands
+* Added `--application-security-groups` argument to `nic [create|ip-config create|ip-config update]`
+* Added `--source-asgs` and `--destination-asgs` arguments to `nsg rule [create|update]`
+* Added `--ddos-protection` and `--vm-protection` arguments to `vnet [create|update]`
+* Added `network [vnet-gateway|vpn-client|show-url]` commands
+
+### Storage
+
+* Fixed issue where `storage account network-rule` commands may fail after updating the SDK
+
+### Eventgrid
+
+* Updated eventgrid SDK to 0.2.0
+
+### SQL
+
+* Changed `sql server list` argument `--resource-group` to be optional. If not specified, all sql servers in the subscription will be returned
+* Added `--no-wait` param to `db [create|copy|restore|update|replica create|create|update]` and `dw [create|update]`
+
+### Keyvault
+
+* Updated azure-keyvault SDK to 0.3.6
```

Comment:
Replace with changes that affect CLI users

Reply:
done